ncm-dpmlfc/src/main/pan/components/dpmlfc/schema.pan

```diff
@@ -75,6 +83,20 @@ function component_dpmlfc_node_config_valid = {
   true;
 };
 
+# Validation of dav parameters
+function component_dpmlfc_dav_config_valid = {
+   if ( is_defined(SELF['DiskFlags']) ) {
+      foreach (i;flag;SELF['DiskFlags']) {
+        if ( !match(flag,'Write|RemoteCopy|NoAuthn') ) return(false);
```

Comment:
strict regexp `^(...)$`
also, maybe use error with message inidicating which key is the problematic one